python/ray/dataframe/dataframe.py

```diff
@@ -459,22 +459,14 @@ def ftypes(self):
         return result
 
     def _correct_dtypes(self):
```

Comment:
Is `_correct_dtypes` the right name to use now? Would `_get_remote_dtypes` be better?

Reply:
Might be better to use `_find_dtypes` since get implies that it performs a blocking `ray.get`